test/Oscoin/Test/Crypto/Blockchain/Block/Generators.hs

```diff
@@ -61,6 +67,36 @@ genStandaloneBlock = do
                }
     pure $ mkBlock header blockDataBeneficiary blockDataTxs
 
+someBeneficiary :: Crypto.HasHashing c => Beneficiary c
+someBeneficiary = Crypto.zeroShortHash
+
+genBeneficiary :: (Hedgehog.MonadGen m, Crypto.HasHashing c) => m (Beneficiary c)
+genBeneficiary = genShortHash
+
+-- | Similar to 'someSealedGenesisBlock\'' with a zero state hash.
+someSealedGenesisBlock
+    :: ( Serialise s
+       , Serialise (Crypto.Hash c)
+       , Crypto.HasHashing c
+       )
+    => s -> Block c tx (Sealed c s)
+someSealedGenesisBlock seal = someSealedGenesisBlock' seal Crypto.zeroHash
+
+-- | Returns an empty genesis block with the given seal and state hash.
+-- The block is not intrinsically valid. It uses 'someBeneficiary'.
+someSealedGenesisBlock' :: (Serialise s, Serialise (Crypto.Hash c), Crypto.HasHashing c) => s -> Crypto.Hash c -> Block c tx (Sealed c s)
+someSealedGenesisBlock' seal stateHash = mkBlock header someBeneficiary []
```

Comment:
For some reason, I find the `Sealed` in the name completely unnecessary, as by default each "valid" block must be sealed -- what about `stockGenesisBlock` & co ? That would go hand in hand with the `stockBeneficiary` idea.

Reply:
👌 on removing “Sealed”.